docs/tutorials/minimal-working-example-react.mdx

```diff
@@ -0,0 +1,394 @@
+---
+title: React
+---
+
+import { ImageWrapper } from "./ImageWrapper";
+import JellyfishArchitecture from "./_jellyfish-architecture.mdx";
+import StartingJellyfishBackend from "./_starting-jellyfish-backend.mdx";
+import StartingJellyfishDashboard from "./_starting-dashboard.mdx";
+
+# React Minimal Working Example
+
+## What you'll learn
+
+This tutorial will guide you through creating your first React project which
+uses Jellyfish client. By the end of the tutorial, you'll have a working web
+application that connects to the front-end dashboard using WebRTC technology,
```

Comment:
maybe link to the jellyfish dashboard, so that users know what we are talking about